src/blocks/recommendations/class-recommendations-block.php

```diff
@@ -0,0 +1,145 @@
+<?php
+/**
+ * Parse.ly Recommendations Block class file.
+ *
+ * @package Parsely
+ * @since 3.2.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely;
+
+/**
+ * Parse.ly Recommendations Gutenberg Block.
+ *
+ * Gutenberg Block that lists Parse.ly recommendations using the /related
+ * Parse.ly API, which is made possible by the Related_API_Proxy endpoint.
```

Comment:
```suggestion
 * Parse.ly Recommendations Block.
```

I don't think there's any value in describing the implementation in the class DocBlock.